src/stores/mainStore.ts

```diff
@@ -55,13 +68,37 @@ export class MainStore {
     );
     this.game.setWinner(winner);
     if (winner === WinnerOptions.Player) {
-      this.player.addPlayerMoney(this.game.playerBet);
+      this.player.addPlayerMoney(this.game.playerBet * 2);
+      // this.player.removePlayerMoney(this.game.bankerBet + this.game.tieBet);
     }
     if (winner === WinnerOptions.Tie) {
-      this.player.addPlayerMoney(this.game.tieBet * 5);
+      this.player.addPlayerMoney(this.game.tieBet * 6);
+      // this.player.removePlayerMoney(this.game.bankerBet + this.game.playerBet);
     }
     if (winner === WinnerOptions.Banker) {
-      this.player.addPlayerMoney(this.game.bankerBet * 0.95);
+      this.player.addPlayerMoney(this.game.bankerBet * 1.95);
+      // this.player.removePlayerMoney(this.game.playerBet + this.game.tieBet);
+    }
+  }
+
+  createSnapshot() {
+    const snapshot = new Snapshot(
+      this.game,
+      this.player,
+      this.game.playerBet,
```

Comment:
But is seems that you already making a snapshot of **game** and **player** store.
Maybe it is worth to simplify?
```typescript
const snapshot = new Snapshot(
      this.game,
      this.player,
    );
```